Sources/Purchasing/OfferingsManager.swift

```diff
@@ -275,6 +275,95 @@ private extension OfferingsManager {
         }
     }
 
+    private func fetchProducts(
+        withIdentifiers identifiers: Set<String>,
+        fromResponse response: OfferingsResponse,
+        completion: @escaping ProductsManagerType.Completion
+    ) {
+        if self.systemInfo.dangerousSettings.uiPreviewMode {
+            let mockProducts = self.createMockProducts(productIdentifiers: identifiers, fromResponse: response)
+            completion(.success(mockProducts))
+        } else {
+            self.productsManager.products(withIdentifiers: identifiers, completion: completion)
+        }
+    }
+
+    // MARK: - Mocks for UI Preview mode
+
+    private func createMockProducts(
```

Comment:
I'm wondering if it makes more sense to name this `createPreviewProducts` and the same for all these changes... Mostly so we don't have "Mocks" in release code, but just "preview" code. Wdyt?

Reply:
Good point! Yes, I like what you suggest